examples/OcclusionSaliency.ipynb

```diff
@@ -124,14 +114,14 @@
     "def app(\n",
     "    image_filepath: str,\n",
     "    # Assuming outputs `nClass` length arrays.\n",
-    "    blackbox_classify: Callable[[PIL.Image.Image], np.ndarray],\n",
+    "    blackbox_classify: Callable[[np.ndarray], np.ndarray],\n",
     "    perturber: PerturbImage,\n",
     "    sal_map_generator: ImageClassifierSaliencyMapGenerator,\n",
     "    fill: Optional[Union[int, Sequence[int]]] = None,\n",
     "    vis_mask_examples: bool = True,\n",
     "):\n",
     "    # Load the image\n",
-    "    ref_image = PIL.Image.open(image_filepath)\n",
+    "    ref_image = np.asarray(PIL.Image.open(image_filepath))\n",
     "    # Use the perterbation API implementation input to generate a bunch of images.\n",
     "    # We will use the outputs here multiple times later so we will just aggregate\n",
     "    # the output here.\n",
```

Comment:
We have this line in the notebook (not sure if it's a carry-over from the PIL-based interface):

```
pert_masks = np.asarray(pert_masks)
```

Is this required since I believe the perturber interface should return np.array based masks already?

Reply:
This is indeed an unnecessary holdover, from the previous streaming interface actually. I'll push a correction momentarily.